### ISSUE TITLE
tceform.php error with typo3 7.6

### ISSUE THREAD

**Issue by jul6art** [tolleiv/TYPO3-EXT-imagemap_wizard#1]
Hi, does anybody make run this extension in back-office? In front, it runs, but impossible to add a new image and configure it in the back-office. Many errors on tceform.php with renderThumbnail method, renderWizards method and getThumbnailScale method. 

Thx for any help

**Comment by garbast** [comment]
We have an EAP project for a complete refactoring if the imagemap_wizard if you are interested please drop me message.

**Comment by jul6art** [comment]
yes, i'm interested by a complete refactoring. Do U have a github of 
this preoject? Thx for your help

**Comment by garbast** [comment]
Yes we have a private repository for the project. Please contact me via sf@marketing-factory.de

**Comment by DavidBruchmann** [comment]
I've a public repository and the backend is working.  
There are still many issues but it is usable, help by reporting or pull-requests is appreciated.  
[DavidBruchmann/imagemap_wizard](https://github.com/DavidBruchmann/imagemap_wizard)
